src/open_inwoner/cms/cases/views/services.py

```diff
@@ -112,17 +117,23 @@ def get_submissions_for_api_group(
 
     def get_submissions(self) -> list[SubmissionWithApiGroup]:
```

Comment:
I notice that for `get_cases` you construct `ZaakWithApiGroup` inside the helper function `_get_raw_cases_for_api_group` which is submitted to the futures. OTOH, for `get_submissions`, the helper function `_get_submissions_for_api_group` only returns the submissions for the relevant api group; the `SubmissionWithApiGroup` is then constructed in the main function `get_submissions`. It would be nice if the fetching of zaken and submissions was uniform as it would make the code easier to read - unless there's a reason for the discrepancy which I've missed.